Dockerfile

```diff
@@ -13,3 +13,11 @@ RUN pip install -r test-requirements.txt
 
 COPY . /src
 RUN pip install .
+
+# install SSHD
+RUN apt-get install -y openssh-client
```

Comment:
better to move this to the top of this stage, so that it doesn't re-install with every code change.

Also needs a apt-get update, --no-install-recommends, and cleaning up after